README.md

```diff
@@ -1 +1,199 @@
-# Catalog-of-my-things
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+<h1> <b> OOP School Library </b> <hr/>
```

Comment:
- [OPTIONAL] The info in the readme should represent the current project. It's marked as optional because it doesn't need to be done in this PR. But we should keep a note or maybe open an extra task for it.